src/components/Sidebar/index.jsx

```diff
@@ -33,7 +32,7 @@ type TSidebarContext = {
 
 export const SidebarContext = React.createContext<TSidebarContext>({
   isOpen: false,
-  toggleSidebar: () => {},
+  toggleSidebar: () => { },
```

Comment:
why add a space here?

Reply:
I'm guessing it was my IDE's formatter, I'll disable it for this project. ESLINT doesn't seem to be throwing errors or warnings for `{}` nor `{ }`. It does if you keep adding spaces, though.

What are we using to enforce formatting style? I see we have ESLINT and quite a few rules for it, but is formatting up to us to do it manually?

The `format` script on `package.json` - which I'm guessing should run Prettier- doesn't seem to be doing anything for me.